actions/Create_groupfolder.bun.ts

```diff
@@ -11,7 +17,7 @@ export async function main(
   ncResource: Nextcloud,
   userId: string | null = null,
   folderName: string,
-  groupName: string,
+  groups: string,
```

Comment:
```suggestion
  groups: Array<{
    id: string,
    key: string,
    type: number,
    displayName: string,
  }>,
```

For some reason the type is not checked in windmill, but this should be the correct one. Not changing functionality :)